app.py

```diff
@@ -1,7 +1,22 @@
-from flask_api import FlaskAPI
+from flask_api import FlaskAPI 
+from flask import request
 
 app = FlaskAPI(__name__)
 
+RESULT = {
+        's1' : 'value1',
+        's2' : 'value2',
+        's3' : 'value3',
+        's4' : 'value4',
+        's5' : 'value5'
+}
+#  quesion 1 finshed  
+@app.route('/data', methods=['GET'])
+def query_string():
+    args = request.args['key']       
+    return RESULT[args]
 
+#  quesion 2 finshed 
```

Comment:
remove irrelevant comments.
We only comment when it comes to application logic that is too hard to understand using plain code